caddy/Caddyfile

```diff
@@ -13,5 +13,7 @@ import endpoint
 
     reverse_proxy /media/* media:8000
 
+    reverse_proxy /prometheus/* prometheus:9090
```

Comment:
Is there some sort of restriction? I don't think, this URL should be public.

Reply:
As part of this setup, no.  My idea is to manage access using HTTP basic auth with the reverse proxy.

Reply:
Ok